pkg/uefi/Dockerfile

```diff
@@ -19,9 +19,9 @@ FROM ${BUILD_CONTAINER} as build
 # hadolint ignore=DL3018
 RUN apk add --no-cache curl make gcc g++ python libuuid iasl nasm util-linux-dev bash git util-linux
 
-RUN git clone --depth 1 -b edk2-stable201908 https://github.com/tianocore/edk2.git /ws
+RUN git clone --depth 1 -b edk2-stable202005 https://github.com/tianocore/edk2.git /ws
 WORKDIR /ws
-RUN git submodule update --init --depth 1
+RUN git submodule update --init
```

Comment:
why did you remove --depth 1 @giggsoff ?

Reply:
With `--depth 1` it fails with:
```error: Server does not allow request for unadvertised object 1cc9cde3448cdd2e000886a26acf1caac2db7cf1
Fetched in submodule path 'UnitTestFrameworkPkg/Library/CmockaLib/cmocka', but it did not contain 1cc9cde3448cdd2e000886a26acf1caac2db7cf1. Direct fetching of that commit failed.
The command '/bin/sh -c git submodule update --init --depth 1' returned a non-zero code: 1
exit status 1
```

So, I follow [the recommendation](https://github.com/tianocore/edk2#submodules) from git.